tests/docker/configs/windows_image.yaml

```diff
@@ -0,0 +1,4 @@
+schemaVersion: 2.0.0
+
+metadataTest:
+  cmd: ["echo bar"]
```

Comment:
Could you add structure tests validating the empty root directories were successfully created?